### PR TITLE
Align byte buffer sizes used for log reading.

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Use DEFAULT_READ_AHEAD_SIZE buffer size in ReadAheadLogChannel by default.
Use buffer with size LOG_HEADER_SIZE in readLogHeader.

### CODE REVIEW

File: community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java [hunk 1]
```diff
@@ -32,13 +32,18 @@
  */
 public class ReadAheadLogChannel implements ReadableVersionableLogChannel
 {
-    public static final int DEFAULT_READ_AHEAD_SIZE = 1024*4;
+    public static final int DEFAULT_READ_AHEAD_SIZE = 1024 * 4;
```
Comment (davidegrohmann) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java]: maybe this can be private?

Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java]: yes good point, it should. Fixed.

File: community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java [hunk 2]
```diff
@@ -32,13 +32,18 @@
  */
 public class ReadAheadLogChannel implements ReadableVersionableLogChannel
 {
-    public static final int DEFAULT_READ_AHEAD_SIZE = 1024*4;
+    private static final int DEFAULT_READ_AHEAD_SIZE = 1024 * 4;
```
Comment (chrisvest) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java:35]: Use `ByteUnit`?

Comment (MishaDemianenko) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/ReadAheadLogChannel.java:35]: yes but not in this PR, migration to ByteUnit will be separate.

**Pull request merged**